conf/server/server_full.conf

```diff
@@ -102,6 +102,19 @@ server {
     # Format of logs, <text|json>. Default: text.
     # log_format = "text"
 
+    # auth_policy_engine: The auth policy engine used for authorization decision
+    # For more details, refer to doc/authorization_policy_engine.md
+    # auth_policy_engine {
```

Comment:
Addressing the other comment together with this... Maybe in that case, perhaps we can just call it

Hmm... only thing that comes to mind here is around when one OPA policy engine fails to evaluate, then there's a fallback policy engine. i.e. remote OPA policy server is down, default back to local OPA policy. 

`auth_opa_policy_engine` and properties of it would be `local` 

```
auth_opa_policy_engine = {
  local {
    policy_path = xyz
    policy_data_path = xyz
  }
}
```